### ISSUE TITLE
Problems in Training Dataset 10049

### ISSUE THREAD

**Issue by Jacker-Li** [yuehuang2023/cryoNeFEN#20]
Hi, I performed data processing and trained the EMPIAR-10049 dataset according to the approach outlined in the paper and the repo.
Here are the commands I used for homogeneous reconstruction:

python train.py \
    /home/li/CyroSPARC/container/CS-10049-1080/J4/J4_005_particles.cs \
    --mask /home/li/CyroSPARC/container/CS-10049-1080/J4/J4_005_volume_mask_refine.mrc \
    --lazy \
    -o /home/li/cryoNeFEN/res/10049/res_homo/loose_mask \
    --split 1 \
    --dim 256 \
    --num-epochs 20 \
    --lr 1e-4

None of the obtained .mrc files could be opened. For example, when attempting to open vol_it000_half1.192.mrc in ChimeraX, the following error occurred:
![Image](https://github.com/user-attachments/assets/0fe8597e-3a5a-42a6-a252-67f4c9e28dda)

Could you explain what might be causing this issue and suggest possible solutions? Your insights would be greatly appreciated. Thank you!

I suspect the issue might stem from the .cs files generated by cryoSPARC. I noticed that the volume dimensions of the trained .mrc files show as (NaN, NaN, NaN). Could you elaborate on your specific processing pipeline for dataset 10049? Thanks!
